### ISSUE TITLE
1904 date system and  ZCL_EXCEL_COMMON=>date_to_excel_string 

### ISSUE THREAD

**Issue by cahp66** [abap2xlsx/abap2xlsx#394]
according to the Microsoft article  https://support.microsoft.com/en-us/kb/180162#/en-us/kb/180162  there are two possible date calculations depending on the settings in the workbook. 

As far as I could see there was no way to determine from the object data whether the 1900 ( the default ) , or the 1904 date system is being used. Although I can't see that this will be a big issue for anyone, as the option to select 1904 is quite well hidden, I though it worth mentioning.

**Comment by StefanSchmoecker** [comment]
OMG
After some searching I actually found the option to change the base-date so this still applies.

But....  That article looks quite old - the latest windows-version mentioned at the end of the article is Excel97 which has been replaced by Excel2000 about 15 years ago :grinning:
Do we have any apple-user here who has created an excel-file and found a problem with our template reader? You might as well look into the _\xl\workbook.xml_ and see if you find an entry 

``` xml
<workbookPr defaultThemeVersion="124226" date1904="1"/>
```

Or did change Microsoft the default-1904-dateformat in the Mac-versions of Excel in the last 10 years.  If they did I don't think we should add a feature here.
